gnd/src/main/java/com/google/android/gnd/model/basemap/Area.java

```diff
@@ -19,46 +19,39 @@
 import com.google.auto.value.AutoValue;
 
 /**
- * An area is a contiguous set of tiles that form a geodesic rectangle.
- * They are uniquely identifiable by their NE and SW bounds (or NW and SE).
+ * An area is a contiguous set of tiles that form a geodesic rectangle. They are uniquely
```

Comment:
Since we won't (and shouldn't?) rely on this as the PK maybe it's not worth mentioning here?